### ISSUE TITLE
Bump cwebp binaries to 1.2.0

### ISSUE THREAD

**Issue by rosell-dk** [rosell-dk/webp-convert#273]

**Comment by rosell-dk** [comment]
New in 1.2.0:

```
- 12/23/2020: version 1.2.0
  * API changes:
    - libwebp:
      encode.h: add a qmin / qmax range for quality factor (cwebp adds -qrange)
  * lossless encoder improvements
  * SIMD support for Wasm builds
  * add xcframeworkbuild.sh, supports Mac Catalyst builds
  * import fuzzers from oss-fuzz & chromium (#409)
  * webpmux: add an '-set loop <value>' option (#494)
  * toolchain updates and bug fixes (#449, #463, #470, #475, #477, #478, #479,
    #488, #491)
```
[[source](https://github.com/webmproject/libwebp/blob/master/NEWS)]

**Comment by rosell-dk** [comment]
PS: I have kept 1.1.0, as 1.2.0 does not run on all systems, see https://github.com/rosell-dk/webp-convert/issues/278
